frontend/Dockerfile

```diff
@@ -1,26 +1,32 @@
 # Frontend dockerfile
-FROM node:14
+# Pinning to 16.14.2 to avoid GH Action lint error exit code 243
+# See: https://stackoverflow.com/a/71892226/3120863
+FROM node:16.14.2
 
 # install dependencies first, in a different location for easier app bind mounting for local development
 # due to default /opt permissions we have to create the dir with root and change perms
 RUN mkdir /opt/node_app && chown node:node /opt/node_app
 WORKDIR /opt/node_app
+
+RUN apt-get update && apt-get install -y make wget \
+    gconf-service gettext-base libasound2 libatk1.0-0 libatk-bridge2.0-0 libc6 \
+    libcairo2 libcups2 libdbus-1-3 libexpat1 libfontconfig1 libgcc1 libgconf-2-4 \
+    libgdk-pixbuf2.0-0 libglib2.0-0 libgtk-3-0 libnspr4 libpango-1.0-0 \
+    libpangocairo-1.0-0 libstdc++6 libx11-6 libx11-xcb1 libxcb1 libxcomposite1 \
+    libxcursor1 libxdamage1 libxext6 libxfixes3 libxi6 libxrandr2 libxrender1 \
+    libxss1 libxtst6 ca-certificates fonts-liberation libappindicator1 libnss3 \
+    lsb-release xdg-utils nano vim procps \
+    && rm -rf /var/lib/apt/lists/*
+
+COPY package*.json ./
 # the official node image provides an unprivileged user as a security best practice
 # but we have to manually enable it. We put it here so npm installs dependencies as the same
 # user who runs the app.
 # https://github.com/nodejs/docker-node/blob/master/docs/BestPractices.md#non-root-user
-
-
-# -- TODO, we should try turning this back on later.
-# USER node
-COPY package.json package-lock.json* ./
+USER node
```

Comment:
we can enable this now, because we switch back to root before running `ENTRYPOINT` below, so the blast radius is a little smaller